### PR TITLE
OCM-11565 | fix: include zero egress status in cluster describe command

### PR DESCRIPTION
**Pull request by philipwu08**
Use tech preview to determine if the describe cluster command should show if a cluster is zero-egress enabled.

### CODE REVIEW

File: cmd/describe/cluster/cmd.go [hunk 1]
```diff
@@ -935,3 +941,21 @@ func getRolePolicyBindings(roleARN string, rolePolicyDetails map[string][]aws.Po
 	}
 	return str, nil
 }
+
+func getZeroEgressStatus(r *rosa.Runtime, cluster *cmv1.Cluster) *string {
+	techPreviewMsg, err := r.OCMClient.GetTechnologyPreviewMessage("hcp-zero-egress", time.Now())
+	if err != nil {
+		r.Reporter.Errorf("Failed to get technology preview message for zero egress: %v", err)
```
[the remark this answers is not in the document]
Comment (philipwu08) [reply cmd/describe/cluster/cmd.go]: Updated, thanks.

**Comment by philipwu08** [comment]
@ciaranRoche Please have another look. Thanks.

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Pull request merged**